spec/utils/test-spliced-sheet.js

```diff
@@ -227,30 +227,24 @@ module.exports = {
         expect(ws.getRow(2).values).to.deep.equal([, 'one', 'two', 'three']);
         expect(ws.getRow(3).values).to.deep.equal([, '2,1', '2,2', '2,3']);
 
-        expect(ws.getRow(3).style).to.deep.equal({
-          alignment: {
-            horizontal: 'left',
-            vertical: 'middle',
-          },
+        expect(ws.getRow(3).style.alignment).to.deep.equal({
```

Comment:
Could you add a test that covers your change?

Reply:
it took o lot of time to find the issue, not sure i'll be able cover with tests coming days